### PR TITLE
Issue #11163: Enforce file size on InputJavadocStyle1

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Resolves update for InputJavadocStyle1 in https://github.com/checkstyle/checkstyle/issues/11163

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java [hunk 1]
```diff
@@ -88,23 +88,53 @@ public void testDefaultSettings()
...
+    public void testDefaultSettingsTwo()
+            throws Exception {
+        final String[] expected = {
+            "61:8: " + getCheckMessage(MSG_UNCLOSED_HTML, "<blockquote> // violation"),
+            "66: " + getCheckMessage(MSG_NO_PERIOD),
+            "98: " + getCheckMessage(MSG_NO_PERIOD),
+        };
+
+        verifyWithInlineConfigParser(
+            getPath("InputJavadocStyleTwo.java"), expected);
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java:107]: InputJavaStyleDefaultSettingsTwo()

**Review by romani — CHANGES_REQUESTED** [review]
Apply to all:

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java (hunk 1 above)
Comment (Rahulkhinchi03) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java]: Changes Done

**Review by romani — APPROVED** [review]

**Comment by Rahulkhinchi03** [comment]
@nick-mancuso https://github.com/checkstyle/checkstyle/pull/11480#issuecomment-1081374069 here

**Pull request merged**